ffi/framebuffer_mxcfb.lua

```diff
@@ -185,13 +183,13 @@ local function mxc_update(fb, update_ioctl, refarea, refresh_type, waveform_mode
     x, y, w, h = bb:getPhysicalRect(x, y, w, h)
 
     if w == 0 or h == 0 then
-        fb.debug("got a 0 size (height and/or width) refresh request, ignoring it.")
+        fb.debug("got an empty (no height and/or width) refresh request, ignoring it.")
         return
     end
 
     if w == 1 and h == 1 then
-        -- avoid system freeze when updating 1x1 pixel block on KPW2 and KV,
-        -- see koreader/koreader#1299 and koreader/koreader#1486
+        -- Avoid a kernel deadlock when updating 1x1 pixel block on KPW2 and KV,
```

Comment:
Huh, I don't think I noticed this particular comment before. But 1 pixel updates could be used to cool effect! ;_;

Reply:
Wouldn't it be better to extend it to 2x2, instead of just dropping it? That standalone pixel . might be important   . :)

Reply:
`1xn` *ought* to work, but `1x1` kills some kernels ^^.

Reply:
The issue is, 2x2 also kills some kernels IIRC :D.

Reply:
Hmm, double-checked, I'm using `(region.width <= 1 || region.height <= 1)` in FBInk.

Reply:
If i remember correctly, in frontend, we always add 1 on each side - so, from KOReader, we should always have larger than 2x2 here. (?)